src/Liip/RMT/Action/CommandAction.php

```diff
@@ -64,4 +72,29 @@ public function execute()
             throw new \RuntimeException("Command [$command] exit with code " . $process->getExitCode());
         }
     }
+
+    /**
+     * Prepares the command.
+     * @param string $command
+     * @return string
+     */
+    protected function prepareCommand($command)
+    {
+        if (substr_count($command, '%') < 2) {
+            return $command;
+        }
+
+        $this->commandVars['new_version'] = Context::getParam('new-version');
+        extract($this->commandVars);
```

Comment:
I didn't know that extract() function. This is no magique. So ugly. So... :puke:
Can we do something more explicit? Like

    foreach(['version', 'new_version'] as $possiblePlaceholder) {
        ...
    }